test/cpp/fluid/mkldnn/test_mkldnn_cpu_quantize_pass.cc

```diff
@@ -0,0 +1,124 @@
+/* Copyright (c) 2023 PaddlePaddle Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include <fstream>
+#include <iostream>
+#include <unordered_map>
+
+#include "test/cpp/inference/api/tester_helper.h"
```

Comment:
why must include this header?

Reply:
I found because this header including other necessary headers, I add them in and remove this header.